### PR TITLE
Fix hardcoded title in WrongPlatformView

### PR DESCRIPTION
**Pull request by vegaro**
We were hardcoding the title in `WrongPlatformView`

simulator_screenshot_11835AB6-6706-43D4-ADE0-51297B7C0247

### CODE REVIEW

File: RevenueCatUI/CustomerCenter/Views/CustomerCenterView.swift [hunk 1]
```diff
@@ -125,6 +125,8 @@ private extension CustomerCenterView {
                     ManageSubscriptionsView(screen: screen,
                                             customerCenterActionHandler: viewModel.customerCenterActionHandler)
                 }
+            } else if let screen = configuration.screens[.management] {
+                WrongPlatformView(screen: screen)
```
Comment (tonidero) [inline RevenueCatUI/CustomerCenter/Views/CustomerCenterView.swift:129]: Hmm are we passing the management screen to the `WrongPlatformView`? Seems wrong to me... As in, it should be a separate screen right?

**Review by tonidero — COMMENTED** [review]
LGTM other than a question

File: RevenueCatUI/CustomerCenter/Views/CustomerCenterView.swift (hunk 1 above)
Comment (vegaro) [reply RevenueCatUI/CustomerCenter/Views/CustomerCenterView.swift:129]: We are not adding another screen to the frontend yet, so reusing that title which is for the "Users with an active subscription", which also applies to this screen.

We should build this screen dynamically as well, and I am thinking we should actually reuse the Management screen for it (filtering out things that don't make sense, like asking for a refund), what do you think? I would do that in another PR though

**Review by tonidero — APPROVED** [review]

File: RevenueCatUI/CustomerCenter/Views/CustomerCenterView.swift (hunk 1 above)
Comment (vegaro) [reply RevenueCatUI/CustomerCenter/Views/CustomerCenterView.swift:129]: Yes it's mainly the custom URLs and the soon to come contact support one which is hardcoded in this screen atm. I don't know what to do with the manage subscriptions button, which is not a path either but doesn't make sense in the regular management screen

**Pull request merged**